assistant/cyton/cyton_controller.py

```diff
@@ -65,15 +68,71 @@ def set_pose(self, q: SE3):
                 self.sock.send(q)
         # TODO: fake printouts
 
-    def go_home(self):
-        self.set_pose(self.robot.qz)
 
-    def grab_object(self, item: BaseItem):
-        robot_object_pose = self.robot.ikine_LM(item.pose, q0=self.pose)
+    def open_gripper(self):
+        """
+        Opens the gripper
+        :return: None
+        """
+        self.robot.open_gripper()
+
+
+    def close_gripper(self):
+        """
+        Closes the gripper
+        :return: None
+        """
+        self.robot.close_gripper()
```

Comment:
So for this I think we need to calculate the distance to close the gripper for. So this function should accept a block object as an input, and we should calculate the correct gripper distance to send based on the block dimensions. Mock pseudo code for this would be like

```python
def close_gripper(self, block):
    width = block.width
    self.set_gripper(width)
```

I think we'll have to play with this in the lab to make sure we're sending the right values for the robot.